ui/hooks/automations.ts

```diff
@@ -39,17 +40,28 @@ export function useListAutomations(namespace = NoNamespace) {
         const kustomizations = (kustRes as ListKustomizationsResponse)
           .kustomizations;
         const helmReleases = (helmRes as ListHelmReleasesResponse).helmReleases;
-
-        return [
-          ..._.map(kustomizations, (k) => ({
-            ...k,
-            kind: FluxObjectKind.KindKustomization,
-          })),
-          ..._.map(helmReleases, (h) => ({
-            ...h,
-            kind: FluxObjectKind.KindHelmRelease,
-          })),
-        ];
+        return {
+          result: [
+            ..._.map(kustomizations, (k) => ({
+              ...k,
+              kind: FluxObjectKind.KindKustomization,
+            })),
+            ..._.map(helmReleases, (h) => ({
+              ...h,
+              kind: FluxObjectKind.KindHelmRelease,
+            })),
+          ],
+          errors: [
+            ..._.map(kustRes.errors, (e) => ({
+              ...e,
+              kind: FluxObjectKind.KindKustomization,
```

Comment:
We might need to do a new type here, above there is a

```
export type Automation = Kustomization & HelmRelease & { kind: FluxObjectKind };
```

So something similar for the `ListError` (in some util / types file I guess)? Otherwise trying to read err.kind might give a type error.